core/xml.ts

```diff
@@ -731,12 +710,8 @@ function applyCommentTagNodes(xmlChildren: Element[], block: Block) {
     const xmlChild = xmlChildren[i];
     const text = xmlChild.textContent;
     const pinned = xmlChild.getAttribute('pinned') === 'true';
-    // AnyDuringMigration because:  Argument of type 'string | null' is not
-    // assignable to parameter of type 'string'.
-    const width = parseInt(xmlChild.getAttribute('w') as AnyDuringMigration);
-    // AnyDuringMigration because:  Argument of type 'string | null' is not
-    // assignable to parameter of type 'string'.
-    const height = parseInt(xmlChild.getAttribute('h') as AnyDuringMigration);
+    const width = parseInt(xmlChild.getAttribute('w') ?? 'NaN', 10);
+    const height = parseInt(xmlChild.getAttribute('h') ?? 'NaN', 10);
```

Comment:
While I think parsing `NaN` is strictly equivalent to the previous code, we probably want to have a default width and height (like 50x50?) instead.